internal/test/round.go

```diff
@@ -30,7 +30,7 @@ func Rounds(rounds []round.Session, rule Rule) (error, bool) {
 		out       = make(chan *round.Message, N*(N+1))
 	)
 
-	if roundType, err = checkAllRoundsSame(rounds); err != nil {
+	if _, err = checkAllRoundsSame(rounds); err != nil {
```

Comment:
@wydengyre I think philosophically we should only change this repo if there's a feature addition or bug-fix.  Refactors should be avoided to keep merge with upstream sane.  If we feel strongly about refactors we should consider doing on upstream and then pulling in here.